pkg/controller/compliancescan/compliancescan_controller.go

```diff
@@ -313,7 +313,8 @@ func (r *ReconcileComplianceScan) phaseRunningHandler(instance *compv1alpha1.Com
 
 	logger.Info("Phase: Running")
 
-	if instance.GetScanType() == compv1alpha1.ScanTypePlatform {
+	switch instance.GetScanType() {
+	case compv1alpha1.ScanTypePlatform:
```

Comment:
If we're already switching to switch-case, what do you think about adding a default and log the wrong scan type? In general I agree that the code shouldn't do "something" unless it knows what the scan is so either if-elseif-else or switch-case is preferable over if-else.

Reply:
nothing unknown should come at this stage, else it'll panic. Validations are done in the pending phase which would prevent this.

Reply:
Thanks, explained on slack